Cockpit 324 (#758)
cockpit-project/cockpit-project.github.io: _posts/2024-09-04-cockpit-324.md

---
title: Cockpit 324
author: jelle
date: '2024-09-04'
tags: podman, files
slug: cockpit-324
category: release

> summary: 'show ports as ranges in podman, viewer/editor for text files'

Reviewer: ```suggestion
summary: 'Show port ranges in Cockpit Podman; view and edit files in Cockpit Files'
```

---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from cockpit-podman 94 and cockpit-files 7:

## Show container/pod ports as ranges

Containers or pods with ports which use the same protocol, host or host ip are now shown as a range if they overlap.

![screenshot of ## show container/pod ports as ranges]({{ site.baseurl }}/images/324-show-container-pod-ports-as-ranges.png)

> ## viewer/editor for text files
> 
> Small text files can now be shown and edited from within the Cockpit-files plugin.
> 
> ![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files.png)
> 
> Viewing a configuration file.
> 
> ![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files-2.png)
> 
> Edit a configuration file.
> 
> ![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files-3.png)
> 
> The editor will show a warning if the file was changed on disk while you where editing it.
> 
> ![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files-4.png)

Reviewer: ```suggestion
## Files: Viewing and editing text files

Cockpit Files now supports viewing and editing small text files, perfect for quickly editing configuration files. Right click a file and select "Open text file" to edit.

![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files.png)

Viewing a configuration file:

![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files-2.png)

Edit a configuration file:

![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files-3.png)

If a file is changed on disk while editing, a warning will appear:

![screenshot of ## viewer/editor for text files]({{ site.baseurl }}/images/324-viewer-editor-for-text-files-4.png)
```
